docs/rules/switch-colon-spacing.md

```diff
@@ -0,0 +1,51 @@
+# Enforce spacing around colons of switch statements (switch-colon-spacing)
+
+Spacing around colons improves readability of `case`/`default` closures.
+
+## Rule Details
+
+This rule reports around colons of `case`/`default` closures if it's different to configured spacing.
```

Comment:
I would rephrase this to "This rule controls spacing around colons of `case` and `default` clauses in `switch` statements."